src/amo/components/InstallWarning/index.js

```diff
@@ -44,12 +43,10 @@ type InternalProps = {|
 |};
 
 export const VARIANT_INCLUDE_WARNING_PROPOSED = 'includeWarning-proposed';
-const WARNING_LINK_DESTINATION =
-  'https://support.mozilla.org/kb/recommended-extensions-program#w_what-are-the-risks-of-installing-non-recommended-extensions';
+const WARNING_LINK_DESTINATION = `${PROMOTED_ADDONS_SUMO_URL}?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=install-warning`;
```

Comment:
possibly a nit for a follow-up issue, but should we define these common utm parameters in a constant/utility function somewhere rather than for each url?

Reply:
We might actually have a function for that somewhere. Let me check...

Reply:
Yep, I changed all of this to use an existing function, plus a new function I created. Thanks for the suggestion!